include/crm/common/output.h

```diff
@@ -207,14 +252,6 @@ struct pcmk__output_fmt_s {
      */
     void (*output_xml) (pcmk__output_fmt_t *out, const char *name, const char *buf);
 
-    /*!
-     * \brief Format a message describing some STONITH event.
-     *
-     * \param out   The output functions structure.
-     * \param event The stonith event to be formatted.
-     */
-    void (*stonith_event) (pcmk__output_fmt_t *out, stonith_history_t *event);
-
```

Comment:
Let's make last_fenced() custom as well; I think pcmk__output_fmt_t should only have generic message functions